cms/bundles/models.py

```diff
@@ -83,6 +86,34 @@ def previewable(self) -> Self:
     def annotate_release_date(self) -> "BundlesQuerySet":
         return self.annotate(release_date=models.F("release_date"))  # type: ignore[no-any-return]
 
+    def annotate_queryset_updated_at(self) -> "BundlesQuerySet":
```

Comment:
This is what the index view does (https://github.com/wagtail/wagtail/blob/60aca9201e78b592fb07227704d8139605afbabb/wagtail/admin/views/generic/models.py#L108-L131)

Added so we surface "updated at" on the dashboard.

I am half-tempted to transform this into its own datetime field on `Bundle`

Reply:
> I am half-tempted to transform this into its own datetime field on Bundle
Would be my preference.

Reply:
ad5663b3